src/Mount.cpp

```diff
@@ -413,23 +413,56 @@ void Mount::configureFocusStepper(byte pin1, byte pin2, int maxSpeed, int maxAcc
 #if RA_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART || DEC_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART                                              \
     || AZ_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART || ALT_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART                                           \
     || FOCUS_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART
-    #if UART_CONNECTION_TEST_TXRX == 1
-bool Mount::connectToDriver(TMC2209Stepper *driver, const char *driverKind)
+    #if UART_CONNECTION_TEST_TXRX == 1 || defined(TEST_VERIFY_MODE)
+bool Mount::connectToDriver(String driverKind, uint16_t *rmsCurrent)
```

Comment:
You can make `driverKind` a const reference (avoids a `copy`):
```suggestion
bool Mount::connectToDriver(const String &driverKind, uint16_t *rmsCurrent)
```